pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/RunArchiver.kt

```diff
@@ -13,7 +13,7 @@ class RunArchiver(packageName: String, project: Project) {
     private val pn: String = packageName
     private val pwd: String? = project.basePath?.let { Paths.get(it).toAbsolutePath().toString() }
     private val rootPiTestDirectoryPathName: Path? = pwd?.let { Paths.get(it, "build", "reports", "pitest") }
-    private val reportDirectory: File = File(Paths.get(rootPiTestDirectoryPathName.toString(), this.pn).toString())
+    private val reportDirectory: File = File(Paths.get(rootPiTestDirectoryPathName.toString(), "test").toString())
```

Comment:
use the `ReportPathGeneratorService` here to get the path to the reports and also put the archive paths into the service